package.json

```diff
@@ -8,6 +8,14 @@
     "type": "git",
     "url": "https://github.com/TNG/momo-scheduler.git"
   },
+  "keywords": [
+    "job",
+    "jobs",
+    "delayed",
+    "scheduler",
+    "task",
+    "recurrent"
```

Comment:
What do you think about `runner` and `interval`?

Reply:
Both sound good to me!